bccsp/pkcs11/pkcs11_test.go

```diff
@@ -283,6 +296,43 @@ func TestECDSAGetKeyBySKI(t *testing.T) {
 	require.Equalf(t, k.SKI(), k2.SKI(), "expected %x got %x", k.SKI(), k2.SKI())
 }
 
+func TestAlternateLabelGeneration(t *testing.T) {
```

Comment:
I'm confused by this test. It's called `TestAlternateLabelGeneration` but I don't see any calls to get attributes to ensure the keys have the appropriate labels. I see the test using the provider to create a key. What's being tested here that wasn't tested in the table test above?